xla/service/gpu/gpu_compiler_test.cc

```diff
@@ -1405,8 +1439,15 @@ TEST_F(PassOrderTest, PassesAreRunInCorrectOrder) {
                   /*last_pass_regex=*/"priority-fusion");
   VerifyPassOrder(/*first_pass_regex=*/"layout-assignment",
                   /*last_pass_regex=*/"layout_normalization");
-  VerifyPassOrder(/*first_pass_regex=*/"host-offload-legalize",
-                  /*last_pass_regex=*/"layout_normalization");
+}
+
+TEST_F(PassOrderTest, OffloadingPassesAreRunInCorrectOrder) {
```

Comment:
Can you explain what the correct order is, at least in a comment?

Also, please reference the bug number that you're trying to work around in the test---since the check that we're not running CSE is testing for a big hack.